integration/size-test/material/list.ts

```diff
@@ -0,0 +1,23 @@
+import {Component, NgModule} from '@angular/core';
+import {platformBrowser} from '@angular/platform-browser';
+import {MatListModule} from '@angular/material/list';
+
+@Component({
```

Comment:
Add a comment explainaing something about using nav-list to make sure that the code for selection-list, etc. isn't included?

Reply:
yeah, will do!